### ISSUE TITLE
Review ADCAD 1.0.0 release

### ISSUE THREAD

**Issue by amoeba** [NCEAS/adc-disciplines#9]
@mbjones asked for a quick check before #7 and #8

**Comment by amoeba** [comment]
Hey @mbjones I found one thing I think we should address before broader distribution: The PLOS alignment uses `http://localhost` URIs everywhere:


```
@prefix plos: <http://localhost/plosthes.2017-1#> .
...
skos:exactMatch <http://localhost/plosthes.2017-1#2668>, wikidata:Q23498 .
```

And the PLOSTHES.ttl file has the same thing:

```
<http://localhost/plosthes.2017-1>
    a :NamedIndividual, skos:ConceptScheme ;
    skos:hasTopConcept <http://localhost/plosthes.2017-1#10580>, <http://localhost/plosthes.2017-1#10670>, <http://localhost/plosthes.2017-1#1969>, <http://localhost/plosthes.2017-1#1990>, <http://localhost/plosthes.2017-1#2884>, <http://localhost/plosthes.2017-1#3625>, <http://localhost/plosthes.2017-1#3917>, <http://localhost/plosthes.2017-1#4019>, <http://localhost/plosthes.2017-1#7094>, <http://localhost/plosthes.2017-1#8400>, <http://localhost/plosthes.2017-1#8539> .
```

This [looks like an upstream issue](https://github.com/PLOS/plos-thesaurus/issues/2). Are we okay with it?

**Comment by mbjones** [comment]
Yes, I looked into it, and that is what PLOSTHES uses in its main source. It is an odd pattern, but technically it only needs to be a unique name. They, of course, will not resolve. But that is how PLOS named their term URIs. So I decided to go with it.

**Comment by mpsaloha** [comment]
Are we using the PLOSTHES verbatim, or modifying a bit? If the latter, I'd
suggest acknowledging them and any other "sources" in an Annotation, and
giving these poor terms a dereferenceable HTTP URI home in our DataONE
ontologies archive- where we can add split/lump,  annotate, axiomatize, and
otherwise revise terms as needed.

cheers,
Mark

**Comment by mbjones** [comment]
We're not "using" the PLOS terms at all, in the sense that we are not defining them. Instead, we are defining an ADCAD term, and then using the PLOS uri as the object of a `skos:exactMatch`. Somewhere in our ontology I want to link our terms to the exact URI used by PLOS. For example, the PLOSTHES class for "Social Sciences" has the term URI `http://localhost/plosthes.2017-1#8539` and can be viewed here: https://bioportal.bioontology.org/ontologies/PLOSTHES/?p=classes&conceptid=http%3A%2F%2Flocalhost%2Fplosthes.2017-1%238539

The corresponding triples I have in ADCAD are:

```ttl
odo:ADCAD_00004
    a owl:Class ;
    rdfs:label "Social Science" ;
    rdfs:subClassOf odo:ADCAD_00000 ;
    skos:exactMatch <http://localhost/plosthes.2017-1#8539>, wikidata:Q34749 .
```

**Comment by amoeba** [comment]
Thanks @mbjones, makes sense to me. Everything looks good to me, then. If/when the PLOS terms get new URIs, we can update our linkage here. I'll subscribe to that issue.

I'm going to leave this ticket open just so the conversation above can continue. Since 1.0.0 is reviewed and tagged, I'm going to push this out since any changes resulting from the discussion here would go in a new version.
